timemachine/fe/single_topology.py

```diff
@@ -1088,18 +1057,22 @@ def _setup_intermediate_chiral_bond_term(self, src_bond, dst_bond, lamb, interpo
 
         return potentials.ChiralBondRestraint(chiral_bond_idxs, chiral_bond_signs).bind(chiral_bond_params)
 
-    def setup_intermediate_state(
-        self,
-        lamb,
-        interpolate_harmonic_bond_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_harmonic_angle_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_periodic_torsion_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_nonbonded_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_chiral_atom_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_chiral_bond_params_fxn: Optional[ParameterInterpolationFxn] = None,
-    ):
+    def setup_intermediate_state(self, lamb, lambda_angles=0.4, lambda_torsions=0.7):
         """
         Setup intermediate states at some value of lambda.
+
+        Parameters
+        ----------
+        lamb: float
+
+        lambda_angles, lambda_torsions: float
+            For ring opening/closing transformations, alchemical parameter values controlling the intervals over which
+            bonds, angles, and torsions are interpolated. Note that these have no effect on terms not involved in ring
+            opening/closing.
+
+            - Bonds are interpolated in the interval [0, lambda_angles]
+            - Angles are interpolated in the interval [lambda_angles, lambda_torsions]
+            - Torsions are interpolated in the interval [lambda_torsions, 1]
```

Comment:
nit[fool-proofing] -- Clearly documented, but would a couple extra assertions to confirm `1 > lambda_torsions > lambda_angles > 0` be useful?

Reply:
Thanks, added in 84a671d